### DISCUSSION TITLE
Log Format issues

### DISCUSSION THREAD

**Discussion by ghost** [teler-sh/teler#136] [Q&A] [answered]
Hello, every time I run the tool I get the same error:

[WRN] No logs analyzed, did you write log format correctly?

I have the tool configured but the problem persists.

[a comment by an automated account is not shown]

**Comment by dwisiswant0** [comment, marked as answer]
As we don't have clear documentation on how to define `log_format`, maybe this can help you: https://github.com/kitabisa/teler/discussions/86#discussioncomment-194019.
